DeviceManager/DeviceHandler.py

```diff
@@ -134,9 +135,30 @@ def get_devices(req):
         init_tenant_context(req, db)
 
         page_number, per_page = get_pagination(req)
-        page = Device.query.paginate(page=page_number,
-                                     per_page=per_page,
-                                     error_out=False)
+        pagination = {'page': page_number, 'per_page': per_page, 'error_out': False}
+
+        parsed_query = []
+        query = req.args.getlist('attr')
```

Comment:
:thinking: 
`req.args` passed on by flask [has this method](http://werkzeug.pocoo.org/docs/0.14/datastructures/#werkzeug.datastructures.MultiDict.getlist), but the simple dict passed on by the hook hasn't.

(this smells like encapsulation leak)